analysis_options.yaml

```diff
@@ -7,13 +7,13 @@ linter:
   rules:
     use_build_context_synchronously: false
     # directives_ordering: true
+    # cascade_invocations: true
 
     annotate_redeclares: true
     avoid_escaping_inner_quotes: true
     avoid_final_parameters: true
     avoid_redundant_argument_values: true
     avoid_types_on_closure_parameters: true
-    cascade_invocations: true
```

Comment:
`cascade_invocations` is nice but also kind of annoying, since it makes it more difficult to move stuff around while you're working.